elastic/kibana
# Correctly sort Index Management data streams by storage size.

Fixes https://github.com/elastic/kibana/issues/86122

## Release note

The list of data streams in Index Management previously sorted alphabetically using the humanized size value. This resulted in an incorrect sort order, for example sorting in ascending order would display a data stream sized at "1kb" before one sized at "5b". Now they're sorted numerically by the raw bytes value, which displays the sorted data streams in the correct order.

## Screenshots

Original incorrect order:

![image](https://user-images.githubusercontent.com/1238659/102411398-bcd30880-3fa6-11eb-8745-84f3376679ce.png)

Current correct order:

![image](https://user-images.githubusercontent.com/1238659/102411406-c2305300-3fa6-11eb-8c61-9232a9d5cd4e.png)

## To test

You can create a data stream with a size in megabytes by following the instructions in the Index Management `README` for creating a data stream and then spamming the request for creating a document in Console by holding down <kbd>Cmd+Enter</kbd>. The Console history will be overwhelmed and you'll see a ton of error toasts but you can ignore those. After you create 800-1000 documents the data stream should be 1mb in size.
